components/org.wso2.carbon.identity.oidc.session/src/main/java/org/wso2/carbon/identity/oidc/session/DefaultOIDCSessionStateManager.java

```diff
@@ -81,6 +84,64 @@ public Cookie addOPBrowserStateCookie(HttpServletResponse response) {
         return cookie;
     }
 
+    /**
+     * Adds the browser state cookie with tenant qualified path to the response.
+     *
+     * @param response
+     * @param request
+     * @param tenantDomain
+     * @return Cookie
+     */
+    @Override
+    public Cookie addOPBrowserStateCookie(HttpServletResponse response, HttpServletRequest request,
```

Comment:
can we deprecate `addOPBrowserStateCookie(HttpServletResponse response)`

Reply:
This method was already deprecated in the interface.